search_gov_crawler/requirements.txt

```diff
@@ -37,7 +37,7 @@ requests==2.28.1
 requests-file==1.5.1
 s3transfer==0.6.0
 Scrapy==2.11.0
-scrapy-playwright==1.45.1
+scrapy-playwright
```

Comment:
Looks like the latest version is 0.0.40.  Should you specify that here?

Reply:
Yes and updated!